pappl/printer-private.h

```diff
@@ -140,6 +140,17 @@ extern void		_papplPrinterWebJobs(pappl_client_t *client, pappl_printer_t *print
 extern void		_papplPrinterWebMedia(pappl_client_t *client, pappl_printer_t *printer) _PAPPL_PRIVATE;
 extern void		_papplPrinterWebSupplies(pappl_client_t *client, pappl_printer_t *printer) _PAPPL_PRIVATE;
 
+extern void		_papplScannerWebCancelAllJobs(pappl_client_t *client, pappl_scanner_t *Scanner) _PAPPL_PRIVATE;
```

Comment:
Hello Sir,
Since there was no scanner.h and scanner.c is empty as of now or any relevant file where I could have added this is to, I added all the changes in printer-private.h. Since Abhik must have created or is working on it, I will write a detailed mail or comment for him to add this to a scanner specific file.

Reply:
Okay, I have shifted the scanner stuff to scanner-private.h, but do we need a scanner.h file as we just added a single scan driver data structure in base.h

Reply:
@Abhik1998 The scanner.h file will define the public API functions/structures for scanners, similar to how printer.h and the various printer*.c files implement the public API for printers.

Reply:
Yes sir, I have updated the scanner.h file